micromamba/tests/test_env.py

```diff
@@ -138,35 +139,39 @@ def test_env_remove(tmp_home, tmp_root_prefix):
 def test_env_update(tmp_home, tmp_root_prefix, tmp_path, prune):
     env_name = "env-create-update"
 
-    # Create env with python=3.11.0 and xtensor=0.25.0
-    helpers.create("python=3.11.0", "xtensor=0.25.0", "-n", env_name, "--json", no_dry_run=True)
+    # Create env with python=3.10.0 and xtensor=0.25.0
+    helpers.create("python=3.10.0", "xtensor=0.25.0", "-n", env_name, "--json", no_dry_run=True)
```

Comment:
That doesnt seem like a trivial change. Is this to reflect conda's behavior?

Reply:
So the changes in the test are a kind of an enhancement containing more checks that the basic previous one:

- It now handles a spec containing a name and a version (it was just the spec name before and there was actually a bug in there)
- Add a new package not already in the current env (the main difference with `conda` and the rational behind this PR), so that it will be installed now.
- Ask to update a spec with a different version (wasn't the case before), so it could be `3.10.0` or anything really, to be updated explicitly to something more recent, `3.11.*` for example...

But that doesn't change the previous test checks generally speaking.

Reply:
> Ask to update a spec with a different version (wasn't the case before), so it could be 3.10.0 or anything really, to be updated explicitly to something more recent, 3.11.* for example...

Would it be worth testing separately 2 or 3 different versions?

Reply:
Hmm yes but it could possibly lead to code duplication, should we rather do that or should I add more comments to explain the test in detail?

Reply:
Maybe add an issue to do that in at a separate time?

Reply:
You mean split the tests (even if it generates code duplication) or add the comments?
I think it's fine if we do it here, while it's fresh ^^ (maybe even best unless we want to change something else in addition to that at the same time in the codebase...)

Reply:
I meant adding an additional "python version" parameter to `test_env_update` and that parameter would be set to 2 or 3 python versions, with at least one being recent, one not being recent, at least. (that's assuming the testing context allows that).
Would it make sense to do that?

Reply:
Ah! No I don't think the versions would make a difference, that's why changing it was not a big deal.

Reply:
I see, ok so nevermind, it's all good 👍🏽

Reply:
Can I have an approval then? :) Thanks!